### PR TITLE
The Career Page 

### PR DESCRIPTION
**Pull request by mmcdevitt1997**
Just check the grammar and spelling. Let me know if I need to add content or fix anything.

### CODE REVIEW

**Comment by shanemiller89** [comment]
:+1:

File: src/career.html [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+      bass guitar for the bands Dogstar and Becky. He has also Acted onstage, he
+      performed as Prince Hamlet for the Manitoba Theatre Centre's production of
+      Hamlet. He has also directed one movie Man of Chi.
+      <!-- List of Skill and talents -->
+    </p>
+
+    <h2>Skills and Talents</h2>
+    <ul>
+      <li>Acting of course</li>
+      <li>Playing base guitar</li>
```
Comment (shanemiller89) [inline src/career.html]: *bass

**Review by shanemiller89 — COMMENTED** [review]
The only thing misspelled I saw was base guitar, it's bass. Other than that, the capitalization in a few areas is strange (like the line "Canadian Movie star, Movie is not capitalized), but as far as coding syntax, I don't see any errors

File: src/career.html [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+      <li>Constantine</li>
+      <div>
+        <img
+          src="img/Devils Advocate.jpg"
+          alt="Devil Advocate Poster"
+          width="400"
+        />
+      </div>
+      <li>The Devil's Advocate</li>
+    </ul>
```
Comment (lindseyemaddox) [inline src/career.html:57]: While I don't think it's anything other than a semantic oddity, I don't think I've ever seen things other than **li** nested inside **ul** tags. There may be a better way to handle this, but I think for our immediate purposes, you're good. Just maybe something to look into in the future.

File: src/career.html [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+</html>
+
+<!-- We want to see the artist's list of talents. This page should include the following components.
+
+Short intro to artist.
+List of skills/talents.
+List of published works - minimum of 3 with images.
+List of places the artist has lived. Be sure to list begin and end times.
+Include any collaborators.
+List of awards. -->
```
Comment (lindseyemaddox) [inline src/career.html:89]: Maybe remove comments before merging?

File: src/career.html (hunk 2 above)
Comment (mmcdevitt1997) [reply src/career.html:57]: You are right I've never seen an image within a list. But the list items correspond with the images that's why I nested them within the list. If the images being in the list have any effect on the style I will move the images out of the list.  👍

File: src/career.html (hunk 1 above)
Comment (mmcdevitt1997) [reply src/career.html]: Those should now be fixed with the new version of this PR. Thank you for the feedback.

**Pull request merged**